templates/blender/index.html

```diff
@@ -0,0 +1,106 @@
+{% extends "blender/_base-blender.html" %}
+
+{% block title %}Blender Support Page{% endblock %}
+{% block meta_description %}SMEs and studios can put the world’s most popular 3D creation software into production with Canonical enterprise-grade support.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1UgPrmv0ambszuLtW9mFjT5KYA7nqaubH3zQg6XNqprw/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-bottomed">
+  <div class="row">
+    <div class="col-6">
+      <h1>Blender Support from&nbsp;Canonical</h1>
+      <p class="p-heading--4">SMEs and studios can put the world’s most popular 3D creation software into production with Canonical enterprise-grade support</p>
+      <p class="p-heading--4">Find out how we can support you</p>
```

Comment:
@anasereijo @davegoddard42 is this OK? "Find out how we can support you" looks odd to me, I think because it's the same size as a heading, and has a lack of punctuation (i.e. a colon "Find out how we can support you:"), it looks like it's just floating there.

![Screenshot from 2021-07-21 15-19-10](https://user-images.githubusercontent.com/2376968/126504478-71d5ec02-7485-4610-b4a5-785e426f8976.png)

Reply:
Yes you're right @sowasred2012, it doesn't look right. It's my fault, I looked at the copy without thinking of the styling in the copy doc. It should have the colon and be body text size.